### PR TITLE
Feature/vr icons

### PR DESCRIPTION
**Pull request by dabeshouse**

### CODE REVIEW

File: core/ide/src/main/java/org/alice/stageide/sceneeditor/viewmanager/MarkerUtilities.java [hunk 1]
```diff
@@ -256,7 +255,10 @@ private static ImageIcon loadIconForObjectMarker(Color color) {
   }
 
   private static Icon loadIconForCameraMarker(Color color) {
-    URL markerIconURL = StorytellingSceneEditor.class.getResource("images/markerIconGrayScale.png");
+    // TOD convert VR flag to parameter
```
Comment (jennej) [inline core/ide/src/main/java/org/alice/stageide/sceneeditor/viewmanager/MarkerUtilities.java:258]: TODO?

Comment (jennej) [reply core/ide/src/main/java/org/alice/stageide/sceneeditor/viewmanager/MarkerUtilities.java:258]: are you just checking if i'm reading the comments? 😂

**Review by jennej — APPROVED** [review]
nice cleanups

File: core/ide/src/main/java/org/alice/stageide/sceneeditor/viewmanager/MarkerUtilities.java (hunk 1 above)
Comment (dabeshouse) [reply core/ide/src/main/java/org/alice/stageide/sceneeditor/viewmanager/MarkerUtilities.java:258]: Would you believe Tod is the next developer to join our team?
I have this addressed in another branch so I will let my typo will stand until then.

Comment (jennej) [reply core/ide/src/main/java/org/alice/stageide/sceneeditor/viewmanager/MarkerUtilities.java:258]: 👍

**Pull request merged**